More responsive text alignment utility classes

bm2u opened the pull request:
- set alignment on certain breakpoints
- avoid alignment on certain breakpoints

Commit:
Responsive alignment
- set alignment on certain breakpoints
- avoid alignment on certain breakpoints

Commit:
Terminate declaration by a semicolon

Commit:
Merge branch 'v4-dev' of https://github.com/twbs/bootstrap into ResponsiveTextAlignment

cvrebert commented on scss/utilities/_text.scss:
```diff
@@ -10,11 +10,25 @@
+      }
+    }
+
+    .text-#{$bp}-not-#{$direction} {
```
Since these don't actually differ by $direction, there's unnecessary duplication.
Why not treat "inherit" like just another direction?

cvrebert commented on scss/utilities/_text.scss:
```diff
@@ -10,11 +10,25 @@
+$text-directions: left center right;
+
+@each $direction in $text-directions {
+  .text-#{$direction} { text-align: #{$direction}; }  // set directions as usual e. g. text-right
```
These deliberately don't exist, since they're equivalent to .text-xs-$direction

cvrebert commented on scss/utilities/_text.scss:
```diff
@@ -10,11 +10,25 @@
+
+  @each $bp in map-keys($grid-breakpoints) {
+    .text-#{$bp}-#{$direction} {
+      @include media-breakpoint-only($bp) {
```
I don't see why you switched from -up to -only.
-up is the more common case, IMHO. And the HTML generally has fewer classes on it with an -up approach.

bm2u commented:
could resolve #19326 from @iamravenous

Commit:
Merge branch 'v4-dev' of https://github.com/twbs/bootstrap into ResponsiveTextAlignment

Commit:
More responsive text alignment utility classes
- revert breakpoints from only to up
- added justify
- only-classes to overrule on certain breakpoints

cvrebert commented on scss/utilities/_text.scss:
```diff
@@ -4,17 +4,26 @@
   @include media-breakpoint-up($breakpoint) {
-    .text-#{$breakpoint}-left   { text-align: left !important; }
-    .text-#{$breakpoint}-right  { text-align: right !important; }
+    .text-#{$breakpoint}-left { text-align: left !important; }
```
Ignore the "!important" warnings.

Commit:
Merge branch 'v4-dev' of https://github.com/twbs/bootstrap into ResponsiveTextAlignment

mdo commented:
Going to punt on the `only` options—we don't do that really anywhere else.
